### ISSUE TITLE
SKSE/Trampoline.h(187): Failed to handle allocation request

### ISSUE THREAD

**Issue by Race533** [ModOrganizer2/modorganizer#1908]
## The problem:
SKSE/Trampoline.h(187): Failed to handle allocation request. NOLVUS is crashing before launching the game. 

## To Reproduce:

1. Download Skyrim NOLVUS mods, set up the game the way you want it for the first time, NOLVUS will run
2. If you uninstall NOLVUS to adjust setting, game options, and try to launch NOLVUS again, you will get this error, no matter how many times you reinstall/trouble shoot. Internet has no known solution.

**Comment by Silarn** [comment]
What steps have you performed to validate that this is a Mod Organizer issue and not the fault of SKSE and / or an SKSE plugin, which it appears to be?

**Comment by Race533** [comment]
I have reinstalled NOLVUS multiple times. It ran the very first time, but only then. I have also run windows repair and resinstalled NOLVUS. Same error every time now.

**Comment by Race533** [comment]
I did not modify the mod collection NOLVUS. I did not upload anything outside its mod collection or changed the launcher / mod organizer. Same error persists.

**Comment by Knit193** [comment]
hei @Race533 , did youb find a solution for the problem? i am running into the same exact problem except not immediately after the 2nd installation ... Usually i run the game from MO2, but today i ran it from Nolvus Dashboard, and now i get the same error ...

**Comment by Race533** [comment]
No, I have not found a solution, I have reset my computer, uninstalled all files related to NOLVUS, started all over again about 4 times, spent 2 days doing this. I gave up, I play now with 208 mods I have selected myself. I check back to NOLVUS around December, 2023, hopefully by then there will be an automated fix for this part of the mod collection. Sorry that I could not help you.

**Comment by Mosesdudu** [comment]
Running into the same problem here, Have tried to reinstall multiple times, even deleted Skyrim and started totally from scratch, Nolvus folder is added to exceptions, tried all the tricks I found on Google (Removing one drive, deleting SKSE from my games folder in documents), nothing helped

**Comment by Mosesdudu** [comment]
Found a fix finally! For me it turns out that it was indeed OneDrive the issue, even though I removed it from my PC, here is what happened:

Right click the documents folder, the one on quick access, go to properties and then location. If you see the path is still OneDrive, try to restore default, if it doesn't work, I followed the steps here:

https://support.microsoft.com/en-gb/topic/operation-to-change-a-personal-folder-location-fails-in-windows-ffb95139-6dbb-821d-27ec-62c9aaccd720 

And it worked finally, no more Trampoline.h(187) error!

**Comment by Specialist456** [comment]
> Found a fix finally! For me it turns out that it was indeed OneDrive the issue, even though I removed it from my PC, here is what happened:
> 
> Right click the documents folder, the one on quick access, go to properties and then location. If you see the path is still OneDrive, try to restore default, if it doesn't work, I followed the steps here:
> 
> https://support.microsoft.com/en-gb/topic/operation-to-change-a-personal-folder-location-fails-in-windows-ffb95139-6dbb-821d-27ec-62c9aaccd720
> 
> And it worked finally, no more Trampoline.h(187) error!

Thank You!
I have always had problems with Bethesda games, years of looking for solutions including uninstalling one drive. You are the first person I have seen recommend this and now everything just works.
